src/content/ConversionHelper/schema.json

```diff
@@ -0,0 +1,27 @@
+[
```

Comment:
tbh, not to sure what the function of schema.json is, can you point me to a primer of where I can read up how this works?

Reply:
The schema defines the interface of the experimental API. It is described in our API documentation:
https://thunderbird-webextensions.readthedocs.io/en/68/how-to/experiments.html